app/scripts/views/mixins/floating-placeholder-mixin.js

```diff
@@ -37,10 +39,20 @@ define(function (require, exports, module) {
       // further conversion will occur.
       if (placeholder !== '') {
         $inputEl.removeAttr('placeholder');
-        $inputEl.prev('.label-helper').text(placeholder).animate( {'top': '-17px'}, 400);
+        $inputEl.prev('.label-helper').text(placeholder).css({'top': '-17px'});
       }
     },
 
+    focusFloatingPlaceholder: function (event) {
```

Comment:
I'm not a fan of passing in the events to functions that do things with the event's currentTarget, the code becomes less generic. I generally use two functions, one which receives the event, and another which receives an element. For example:

```js
events: {
  'blur input': 'onInputBlur',
 ...
},

onInputBlur: function (event) {
  this.unfocusFloatingPlaceholder($(event.currentTarget));
},

unfocusFloatingPlaceholder($inputEl) {
  $inputEl.prev('.label-helper').removeClass('focused');
},
...
```

I find separating the concerns like this makes testing easier and it means the functions that work with the elements can be used elsewhere without passing in an event.